### ISSUE TITLE
Add classification type to more entity types

### ISSUE THREAD

**Issue by lilleyse** [CesiumGS/cesium#6661]
`classificationType` is only available on entity types than can clamp to ground (polygons, rectangles, etc), but it would great to see this for all types (boxes, polyline volumes, and others). It's possible from the Primitive API layer ([Classification demo](https://cesiumjs.org/Cesium/Build/Apps/Sandcastle/?src=Classification.html)) and would be convenient to add to the entity API.

@bagnell do you know if there are any technical roadblocks to supporting this? It might be a good issue for @loshjawrence to check out during down time while getting asset pipeline development set up.

**Comment by hpinkos** [comment]
@lilleyse Are you sure this isn't already supported? It looks like in that Sandcastle example you linked to, ClassificationType is used for an `EllipsoidGeometry`:
```js
var treeHighlight1 = scene.primitives.add(new Cesium.ClassificationPrimitive({
    geometryInstances : new Cesium.GeometryInstance({
        geometry : new Cesium.EllipsoidGeometry({
            radii : new Cesium.Cartesian3(3.25, 5.0, 4.0)
        }),
        modelMatrix : modelMatrix,
        attributes : {
            color : Cesium.ColorGeometryInstanceAttribute.fromColor(Cesium.Color.fromCssColorString('#F26419').withAlpha(0.5)),
            show : new Cesium.ShowGeometryInstanceAttribute(true)
        },
        id : 'volume 1'
    }),
    classificationType : Cesium.ClassificationType.CESIUM_3D_TILE
}));
```

**Comment by hpinkos** [comment]
@lilleyse we don't have the concept of a "Classification" at the entity level at all as far as I know.  We use `GroundPrimitive` for `corridor`, `ellipse`, `polygon` and `rectangle` and use `Primitive` for all other types.

**Comment by mramato** [comment]
@hpinkos all of the Entity XXXGraphics that can be clamped to ground have a `classificationType` property.  What @lilleyse is saying is that BoxGraphics and other volumes that don't clamp to ground should still have `classificationType` as well because they can classify 3D tilesets (as in the linked demo).

**Comment by hpinkos** [comment]
@mramato yeah, I get that, but we would want a way to determine when to use `ClassificationPrimitive` and when to use `Primitive`.  I guess it would just be when `classificationType` is defined?

**Comment by mramato** [comment]
I see where you're coming from now.  Yes we would probably have to use it depending on if it's defined or not.  Can `ClassificationPrimitive` batched?

This is probably a more complicated issue than @lilleyse had in mind for onramping.

**Comment by lilleyse** [comment]
> This is probably a more complicated issue than @lilleyse had in mind for onramping.

Sounds like it. Thanks for the extra details @hpinkos @mramato.
